### PR TITLE
Fader Networks Manipulating Images by Sliding Attributes

### PR DESCRIPTION
**Pull request by nathanpainchaud**

### CODE REVIEW

File: article/_posts/2020-11-20-FaderNetworks.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+![](/article/images/FaderNetworks/figure5.jpg)
+
+
+# Remarks
+- In theory, Fader networks should be able to handle continuous data attributes by switching the target of the
+adversarial component, and adapting it's loss from cross-entropy to some kind of regression loss (e.g. MAE, MSE, etc.).
+However, in practice, the method is known to perform poorly on non-categorical data attribute[^1].
+
+- In practice, the addition of an adversarial network makes the training much less stable, as could be expected. Thus,
+the author describe (common) implementation details that they required for their method to work, e.g. dropout,
```
Comment (jhlegarreta) [inline article/_posts/2020-11-20-FaderNetworks.md:91]: authors
or
the authors

File: article/_posts/2020-11-20-FaderNetworks.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+![](/article/images/FaderNetworks/table1.jpg)
+
+![](/article/images/FaderNetworks/figure3.jpg)
+
+![](/article/images/FaderNetworks/figure5.jpg)
+
+
+# Remarks
+- In theory, Fader networks should be able to handle continuous data attributes by switching the target of the
+adversarial component, and adapting it's loss from cross-entropy to some kind of regression loss (e.g. MAE, MSE, etc.).
```
Comment (jhlegarreta) [inline article/_posts/2020-11-20-FaderNetworks.md:87]: its loss

File: article/_posts/2020-11-20-FaderNetworks.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+## Discriminator objective
+The discriminator tries to predict the attributes of the input image given its latent representation according to:
+
+$$
+\mathcal{L}_{\text{dis}}(\theta_{\text{dis}}|\theta_{\text{enc}}) = 
+-\frac{1}{m} \sum_{(x,y) \in D} \text{log} P_{\theta_{\text{dis}}}(y | E_{\theta{\text{enc}}}(x))
+$$
+
+## Adversarial objective
+The encoder tries to learn that optimizes the two objectives mentioned above: namely a good reconstruction but a bad
```
Comment (jhlegarreta) [inline article/_posts/2020-11-20-FaderNetworks.md:44]: something is missing here: "(...) to learn that (...)" or maybe "that" is not necessary.

File: article/_posts/2020-11-20-FaderNetworks.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+# Highlights
+- New encoder-decoder architecture that disentangle salient information in images and attributes directly in the latent
+space;
+- At test-time, using continuous attribute values allows to adjust how much of an attribute is perceivable in the
+generated image.
+
+
+# Introduction
+The authors' goal is to control some attributes of interest images, for which transformations are ill-defined and
+training is unsupervised, i.e. no examples of images before and after the transformations are available. They achieve
```
[the remark this answers is not in the document]
Comment (nathanpainchaud) [reply article/_posts/2020-11-20-FaderNetworks.md]: Yeah, reading that again I can see that what I mean by "transformation" is not really clear if you're not familiar with the context. I've added an alternate explanation, that should give a better understanding of what a "transformation" is supposed to be. Let me know if it's still unclear for you!

[the remark this answers is not in the document]
Comment (jhlegarreta) [reply article/_posts/2020-11-20-FaderNetworks.md]: It is more clear now. Thanks.

**Review by jhlegarreta — APPROVED** [review]

**Pull request merged**